BookingsAPI/Bookings.AcceptanceTests/Steps/ParticipantsSteps.cs

```diff
@@ -118,11 +118,6 @@ public void ThenIndividualDetailsShouldBeUpdated(string participant)
             model.DisplayName.Should().Be(updateParticipantRequest.DisplayName);
             model.TelephoneNumber.Should().Be(updateParticipantRequest.TelephoneNumber);
             if (!participant.Equals("Representative")) return;
```

Comment:
You can get rid of that if statement as well

Reply:
Oh yes,I missed it here

Reply:
Done